web/templates/index.html

```diff
@@ -19,6 +19,12 @@
   <link rel="stylesheet" href="/static/css/font-awesome.min.css"/>
 </head>
 <body>
+  <script src="https://cdn.ravenjs.com/2.3.0/raven.min.js"></script>
+  <script>
+    // Setup sentry reporting
+    Raven.config('https://fd20ee7ca8f7421fb4cddbd489117f22@app.getsentry.com/73224').install()
```

Comment:
I was also curious if there was any way to bundle this with the rest of our app.